templates/zerver/help/mention-a-team-member.md

```diff
@@ -0,0 +1,11 @@
+#Mention a Team Member
+
+![Mention a Team Member](/static/images/help/mention_user.png)
+
+When having a conversation in Zulip, it is sometimes necessary to call the attention of another member. You can ping ("@mention") other users with a particular stream-message from within the body of the message itself. Type @ and the beginning of their email address or one of their names, and the system will offer you tab-completions. The completed @mention will look something like: @\*\*name\*\*, and it will appear as @name in your posted message.

```

Comment:
Baa, the comments are also markdown. I recommend surrounding the @mention lines in back ticks so they appear like code blocks.